### PR TITLE
[HOPSWORKS-3202] Extra filter user guide

### PR DESCRIPTION
**Pull request by kennethmhc**

### CODE REVIEW

File: docs/user_guides/fs/feature_view/training-data.md [hunk 1]
```diff
@@ -2,52 +2,71 @@
 
 Training data can be created from the feature view and used by different ML libraries for training different models.
 
-You can read [training data concepts](../../../concepts/fs/feature_view/offline_api.md) for more details.
+You can read [training data concepts](../../../concepts/fs/feature_view/offline_api.md) for more details. To see a full example of how to create a training data, you can read [this notebook](https://github.com/logicalclocks/hopsworks-tutorials/blob/master/fraud_batch/2_feature_view_creation.ipynb).
```
Comment (tdoehmen) [inline docs/user_guides/fs/feature_view/training-data.md:5]: "create a training dataset", or "create training data"

File: docs/user_guides/fs/feature_view/training-data.md [hunk 2]
```diff
@@ -2,52 +2,71 @@
...
+    description = 'transactions fraud batch training dataset',
     data_format = 'csv',
     write_options = {"wait_for_job": False}
 ) # By default, it is materialised to HopsFS
 print(job.id) # get the job's id and view the job status in the UI
 ```
 
+
+### Extra filters
+Sometimes data scientists need to train different models using subsets of a dataset. For example, there can be different models for different countries, seasons and different groups. One way is to create different feature views for training different models. Another way is to add extra filters on top of the feature view when creating a training data.
```
Comment (tdoehmen) [inline docs/user_guides/fs/feature_view/training-data.md:27]: different countries, seasons**,** and different groups.

Comment (tdoehmen) [reply docs/user_guides/fs/feature_view/training-data.md:27]: "creating a training data." see above

**Comment by kennethmhc** [comment]
@tdoehmen addressed your comment.

**Review by tdoehmen — APPROVED** [review]

**Pull request merged**